fluent/handler.py

```diff
@@ -136,13 +136,15 @@ def __init__(self,
                  port=24224,
                  timeout=3.0,
                  verbose=False,
-                 buffer_overflow_handler=None):
+                 buffer_overflow_handler=None,
+                 nanosecond_precision=False):
```

Comment:
why not just use `**kwargs` for everything but `tag` (since you need to store it on `self`) as a catch-all pass-through mechanism. That way you don't have to keep editing this function every time the `FluentSender` interface changes?